processor/src/errors.rs

```diff
@@ -90,8 +110,10 @@ pub enum ExecutionError {
         #[source_code]
         source_file: Option<Arc<SourceFile>>,
         clk: RowIndex,
-        err_code: Felt,
```

Comment:
Removing `err_code` would mean that once debug info is stripped from a program, we won't be able to identify errors, right? Basically, right now we have:

1. If the debug info is available, we can print out the full error message.
2. If the debug info is not available, we can print out error code.

If I'm understanding correctly, after this PR, the second option is no longer available. Maybe that's fine, but I just want to make sure we are not doing this by mistake.

Reply:
Good point. I think to retain some chance of identifying errors in a production environment, having at least the error code would be good.